Shell/Options.cpp

```diff
@@ -1935,8 +1948,16 @@ void Options::init()
     _clausificationOnTheFly.addProblemConstraint(hasHigherOrder());
     _clausificationOnTheFly.tag(OptionTag::HIGHER_ORDER);
 
-    _casesSimp = BoolOptionValue("cases_simp", "cs", false);
-    _casesSimp.description =
+    _equalityToEquivalence = BoolOptionValue("equality_to_equiv","e2e",false);
+    _equalityToEquivalence.description=
+      "Equality between boolean terms changed to equivalence \n"
+      "t1 : $o = t2 : $o is changed to t1 <=> t2";
+    _lookup.insert(&_equalityToEquivalence);
+    // potentially could be useful for FOOL, so am not adding the HOL constraint
+    _equalityToEquivalence.tag(OptionTag::HIGHER_ORDER);
```

Comment:
I think we should either drop the `tag` or the comment. Do you agree? (I assume this is a preprocessing rule?)